schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -985,8 +1014,15 @@
         },
         "source": {
             "type": "object",
-            "description": "This is the source information (who discovered it, who researched it, etc.) and optionally a chain of CNA information (e.g. the originating CNA and subsequent parent CNAs who have processed it before it arrives at the MITRE root).\n Must contain: IF this is in the root level it MUST contain a CNA_chain entry, IF this source entry is NOT in the root (e.g. it is part of a vendor statement) then it must contain at least one type of data entry.\n Mandatory in: none | Optional in: all containers",
+            "description": "This is the source information (who discovered it, who researched it, etc.) and optionally a chain of CNA information (e.g. the originating CNA and subsequent parent CNAs who have processed it before it arrives at the MITRE root).\n Must contain: IF this is in the root level it MUST contain a CNA_chain entry, IF this source entry is NOT in the root (e.g. it is part of a vendor statement) then it must contain at least one type of data entry.",
             "minProperties": 1
+        },
+        "language": {
+            "type": "string",
+            "description": "BCP 47 language code",
+            "default": "en",
+            "minLength": 1,
+            "maxLength": 36
```

Comment:
Is there a pattern for this?

Reply:
https://tools.ietf.org/html/bcp47

Reply:
for language-region match
^[[:alpha:]]{2,3}(?:$|-[[:alnum:]]{2,3}$)

found full regex examples, but they do not appear to work as as a string in javascript or JSON
https://stackoverflow.com/questions/7035825/regular-expression-for-a-language-tag-as-defined-by-bcp47
https://schneegans.de/lv/

Reply:
Completing this will address #18.